docs/tutorial.md

```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
 8. Select __Paint Brush__ to color in the entire burger. You can enlarge the size of the __Paint Brush__ in the __Tools Settings Bar__ to speed up the process.
-9. Select __Eraser__ to clean up any color that doesn't belong. 
-10. Save your progress and it should look like this: 
+9. Select the [__Eraser__ tool](remove.md#eraser) to clean up any color that doesn't belong. 
+10. Save your progress and it should look something like this: 
 ![color bun](img/burger/color_bun.gif)
 
 ### __Cheese, Tomatoes & Lettuce__ ###
 We'll now work on the cheese, tomatoes, and lettuce:
 
 1. Create a new layer. Make sure this new layer is *below* __Ink__ and *above* __Bun Color__ layers.
-2. Open __Layer Properties__ for the new layer and enter __Other Color__ in the text box. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Other Color__. 
 3. Select __OK__ to close the window. 
-4. Select __Other Color__ to make it an active layer. 
+4. Select __Other Color__ layer to make it an active layer. 
 5. Using the same concept as the bun, select __Color Picker__ to pick the colors from the reference image to apply to the project for the cheese, tomatoes, and lettuce. 
-6. Select __Paint Brush__ and __Eraser__ to color in the appropriate areas. 
+6. Select the __Paint Brush__ and __Eraser__ tools to color in the appropriate areas. 
 7. Save your progress and it should look like this: 
 ![other color](img/burger/other_color.png)
 
 ### __Patty__ ###
 For the patty, we want to create a gradient to replicate the grilled side of the patty:
 
 1. Create a new layer. Make sure this new layer is *below* __Ink__ and *above* __Other Color__ layers.
-2. Open __Layer Properties__ for the new layer, enter __Patty Color__, and enter opacity to 90. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Patty Color__ and enter an **Opacity** value of 90. 
 3. Select __OK__ to close the window. 
-4. Select __Patty Color__ to make it an active layer. 
+4. Select __Patty Color__ layer to make it an active layer. 
 5. Using the same concept as earlier, select __Color Picker__ to pick the patty color from the reference image and switch back to the project. 
 6. Right-click the black color in the __Palette__ to set the secondary color and select __Gradient__. 
 7. Click and hold on the drawing in any area while moving the cursor in different directions to form a gradient. Let go once you're satisfied with the way the color is distributed. Repeat if necessary.
 8. Select __Eraser__ to erase anything that is outside of the patty. Resize __Eraser__ to get the required size.
-9. Double-click on __Patty Color__ and enter opacity to 100. Select __OK__ to close the window. 
+9. Double-click on __Patty Color__ layer in the **Layers** window and enter **Opacity** value of 100. Select __OK__ to close the window. 

```

Comment:
keep original because this is too wordy

Reply:
opacity to 100 does not make sense grammatically

Reply:
oh nvm i agree. i don't know why i wrote i was keeping the original. maybe i got mixed up with another line. here. this part is good.